JavaCryptographicArchitecture/src/PKIXParameters.crysl

```diff
@@ -7,11 +7,14 @@ OBJECTS
 EVENTS
 	c1: PKIXParameters(keyStore);
 	c2: PKIXParameters(trustAnchors);
+	Cons := c1 | c2;
+	
 ORDER
-	c1
+	Cons
 
 REQUIRES
 	generatedKeyStore[keyStore];		
-	//generatedTrustAnchor[];							
+	//generatedTrustAnchor[];	
```

Comment:
Why is this one commented?

Reply:
The PKIXParamters constructor recieves a  TrustAnchor set. This set needs to be created manually. Because there is no rule that ensured a predicate for a complete set of TrustAnchor, the comment can be deleted. https://github.com/CROSSINGTUD/Crypto-API-Rules/pull/65/commits/6de9d3516dd4e7603978c0eb677ebe3d88ff970c

Reply:
Shouldn't we rather create the rule to make this predicate work?

Reply:
I couldn't find any class which returns a Set of TrustAnchor objects..this is what I meant by "This set needs to be created manually" ..the developer needs to add all created TrustAnchor by hand to a Set... this ensures at least that the individual objects have the predicate through the TrustAnchor rule

Reply:
Ah, I see. Sounds good to me then.